### PR TITLE
Update auth-backend to 2.0.1

### PR DESCRIPTION
**Pull request by awaisdar001**
This PR updates the `auth-backends` to the latest. 
I have run `make upgrade` command to keep any the packages in sync. The package `auth-backend` was not pinned so it was upgraded to `2.0.1` automatically when I run the command. 

I am tagging @jmbowman as he has more context of using OEP-18.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -147,13 +148,6 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
 	pip-compile --upgrade -o requirements/production.txt requirements/production.in
 	pip-compile --upgrade -o requirements/test.txt requirements/test.in
-	scripts/post-pip-compile.sh \
```
Comment (jmbowman) [inline Makefile:150]: You should also be able to delete `scripts/post-pip-compile.sh` from the repository.

File: requirements/base.txt [hunk 2]
```diff
@@ -24,7 +24,7 @@ cssutils==1.0.2           # via premailer
 defusedxml==0.6.0         # via zeep
 django-appconf==1.0.3     # via django-compressor
 django-compressor==2.2
-django-cors-headers==2.5.3
+django-cors-headers==3.0.1
```
Comment (jmbowman) [inline requirements/base.txt:28]: It looks like this upgrade will require a minor update to the `CORS_ORIGIN_WHITELIST` setting: https://github.com/ottoyiu/django-cors-headers/blob/master/HISTORY.rst#300-2019-05-10

Comment (awaisdar001) [reply requirements/base.txt:28]: Thank you @jmbowman

**Comment by awaisdar001** [comment]
@jmbowman I have started getting test failures on the change 
```_validate_required_claims
    raise MissingRequiredClaimError('exp')
MissingRequiredClaimError: Token is missing the "exp" claim
```

I am not sure how to fix them, I do not know much about this repo. Do you have some ideas?

**Comment by awaisdar001** [comment]
I am able to figure out the issue as I sat with @saleem-latif .
I am intentionally going to pin `edx-drf-extensions==2.2.0` and `edx-rbac==0.2.0` as the work is still in progress for them.

**Comment by awaisdar001** [comment]
FYI @jmbowman Please review.

**Comment by awaisdar001** [comment]
@noraiz-anwar Please take a look.

**Review by noraiz-anwar — APPROVED** [review]
👍

**Review by jmbowman — APPROVED** [review]
Looks good!

**Pull request merged**